app/assets/javascripts/select-kit/components/tag-drop.js.es6

```diff
@@ -1,9 +1,10 @@
 import ComboBoxComponent from "select-kit/components/combo-box";
 import DiscourseURL from "discourse/lib/url";
+import Tags from "select-kit/mixins/tags";
```

Comment:
Let's name this `TagsMixin` instead since most of the other files follow this convention.